smartsim/database/orchestrator.py

```diff
@@ -254,22 +252,10 @@ def __init__(
             **kwargs,
         )
 
-        # detect if we can find at least the redis binaries. We
-        # don't want to force the user to launch with RedisAI so
-        # it's ok if that isn't present.
         try:
-            # try to obtain redis binaries needed to launch Redis
-            # will raise SSConfigError if not found
-            self._redis_exe  # pylint: disable=W0104
-            self._redis_conf  # pylint: disable=W0104
             CONFIG.database_cli  # pylint: disable=W0104
         except SSConfigError as e:
-            raise SSConfigError(
-                "SmartSim not installed with pre-built extensions (Redis)\n"
-                "Use the `smart` cli tool to install needed extensions\n"
-                "or set REDIS_PATH and REDIS_CLI_PATH in your environment\n"
-                "See documentation for more information"
-            ) from e
+            ...
```

Comment:
Shouldn't matter too much because this module will either be heavily revised or outright removed in future, but just for posterity's sake: consider removing the `try/execpt` block entirely just to not accidentally suppress an exception

Reply:
removed